src/pages/Environments.js

```diff
@@ -466,20 +466,20 @@ const Environments = () => {
             }
           },
           {
```

Comment:
Not directly related to the ticket. 
The diff below can be misleading. There is no logic change. I merely swapped the positions of the `Status` and `Location` columns for `Your Disks` table so it's consistent with `Your Cloud Environments` table. However, git doesn't seem to have realized that.

**Old:**
![image](https://user-images.githubusercontent.com/5438223/133612756-725a42e4-3490-4acf-92f6-3d5ffbd9410c.png)

**New:**
![image](https://user-images.githubusercontent.com/5438223/133612799-9a1126c4-d37e-4abb-b497-00b775deacf8.png)

Reply:
Nice 👍